### PR TITLE
Add description for kcn, kpn, ken, kscn, kspn, ksen and kgen

### PR DESCRIPTION
**Pull request by andybclee**
## Proposed changes

- This PR adds description on doc.go files fir kcn, kpn, ken, kscn, kspn, ksen and kgen 
- All those packages are for defining command line options and node types

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by jeonghonah** [comment]
> IMHO, Because, this description from `kXXX help`. It can be changed.
> Every time, we should update this docs. I think this is not good for maintenance.

I agree with the opinion. It will be nice if we automatically generate the description form the help.

**Comment by andybclee** [comment]
@KimKyungup @jeonghonah Applied your suggestion. Please take another look =)

File: cmd/kcn/doc.go [hunk 1]
```diff
@@ -18,5 +18,14 @@
...
-// kcn is the command-line client for Klaytn Consensus Node.
+/*
+kcn is the command-line client for Klaytn Consensus Node.
+
+kcn has the node type of "cn" internally. Please try `kcn -h` to see commands and options list.
+
+Source Files
+
+Each file contains following contents
+ - main.go : Defines available options and initialize the application with given options
```
Comment (aidan-kwon) [inline cmd/kcn/doc.go]:
```suggestion
 - main.go : Defines available options and initializes the application with given options
```

File: cmd/ken/doc.go [hunk 2]
```diff
@@ -18,5 +18,14 @@
...
-// ken is the command-line client for Klaytn Endpoint Node.
+/*
+ken is the command-line client for Klaytn Endpoint Node.
+
+ken has the node type of "en" internally. Please try `ken -h` to see commands and options list.
+
+Source Files
+
+Each file contains following contents
+ - main.go : Defines available options and initialize the application with given options
```
Comment (aidan-kwon) [inline cmd/ken/doc.go]:
```suggestion
 - main.go : Defines available options and initializes the application with given options
```

File: cmd/kgen/doc.go [hunk 3]
```diff
@@ -14,5 +14,17 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
 
-// kgen can be used to generate the associated nodekey information.
+/*
+kgen can be used to generate a nodekey and derived address and kni information.
```
Comment (aidan-kwon) [inline cmd/kgen/doc.go]:
```suggestion
kgen can be used to generate a nodekey and a derived address and kni information.
```

File: cmd/kgen/doc.go [hunk 4]
```diff
@@ -14,5 +14,17 @@
...
+
+By providing some option in the command line, an user can assign ip and port for kni.
```
Comment (aidan-kwon) [inline cmd/kgen/doc.go]:
```suggestion
By providing some options in the command line, a user can assign IP and port for kni.
```

File: cmd/kgen/doc.go [hunk 5]
```diff
@@ -14,5 +14,17 @@
...
+
+Options
+
+All available options are as follows.
+   --file        Generate a nodekey and a Klaytn node information as files
+   --ip value    Specify an ip address (default: "0.0.0.0")
```
Comment (aidan-kwon) [inline cmd/kgen/doc.go]:
```suggestion
   --ip value    Specify an IP address (default: "0.0.0.0")
```

File: cmd/kgen/doc.go [hunk 6]
```diff
@@ -14,5 +14,17 @@
...
+   --port value  Specify a tcp port number (default: 32323)
+   --help, -h    show help
```
Comment (aidan-kwon) [inline cmd/kgen/doc.go]:
```suggestion
   --help, -h    Show help
```

File: cmd/kpn/doc.go [hunk 7]
```diff
@@ -18,5 +18,14 @@
...
-// kpn is the command-line client for Klaytn Proxy Node.
+/*
+kpn is the command-line client for Klaytn Proxy Node.
+
+kpn has the node type of "pn" internally. Please try `kpn -h` to see commands and options list.
+
+Source Files
+
+Each file contains following contents
+ - main.go : Defines available options and initialize the application with given options
```
Comment (aidan-kwon) [inline cmd/kpn/doc.go]:
```suggestion
 - main.go : Defines available options and initializes the application with given options
```

File: cmd/kscn/doc.go [hunk 8]
```diff
@@ -18,5 +18,14 @@
...
-// kscn is the command-line client for Klaytn ServiceChain Node.
+/*
+kscn is the command-line client for Klaytn ServiceChain Node.
+
+kscn has the node type of "cn" internally. Please try `kscn -h` to see commands and options list.
+
+Source Files
+
+Each file contains following contents
+ - main.go : Defines available options and initialize the application with given options
```
Comment (aidan-kwon) [inline cmd/kscn/doc.go]:
```suggestion
 - main.go : Defines available options and initializes the application with given options
```

File: cmd/ksen/doc.go [hunk 9]
```diff
@@ -18,5 +18,14 @@
...
-// ksen is the command-line client for Klaytn ServiceChain Endpoint Node.
+/*
+ksen is the command-line client for Klaytn ServiceChain Endpoint Node.
+
+ksen has the node type of "en" internally. Please try `ksen -h` to see commands and options list.
+
+Source Files
+
+Each file contains following contents
+ - main.go : Defines available options and initialize the application with given options
```
Comment (aidan-kwon) [inline cmd/ksen/doc.go]:
```suggestion
 - main.go : Defines available options and initializes the application with given options
```

File: cmd/kspn/doc.go [hunk 10]
```diff
@@ -18,5 +18,14 @@
...
-// kspn is the command-line client for Klaytn ServiceChain Proxy Node.
+/*
+kspn is the command-line client for Klaytn ServiceChain Proxy Node.
+
+kspn has the node type of "pn" internally. Please try `kspn -h` to see commands and options list.
+
+Source Files
+
+Each file contains following contents
+ - main.go : Defines available options and initialize the application with given options
```
Comment (aidan-kwon) [inline cmd/kspn/doc.go]:
```suggestion
 - main.go : Defines available options and initializes the application with given options
```

File: cmd/kscn/doc.go [hunk 11]
```diff
@@ -18,5 +18,14 @@
 // This file is derived from cmd/geth/main.go (2018/06/04).
 // Modified and improved for the klaytn development.
 
-// kscn is the command-line client for Klaytn ServiceChain Node.
+/*
+kscn is the command-line client for Klaytn ServiceChain Node.
```
Comment (jeonghonah) [inline cmd/kscn/doc.go]:
```suggestion
kscn is the command-line client for Klaytn ServiceChain Consensus Node.
```

**Pull request merged**